google-pubsub/1.0/modules/ROOT/pages/google-pubsub-connector-reference.adoc

```diff
@@ -605,7 +620,25 @@ This type enables you to set the acknowledgment state of messages in an existing
 | Field | Type | Description | Default Value | Required
 | Name a| String | Name of the snapshot. |  |
 | Topic a| String | Name of the topic from which the snapshot retains messages. |  |
-| Expire Time a| DateTime | Time that the snapshot is guaranteed to exist up until. A newly-created snapshot expires no later than seven days from the time of its creation. Its exact lifetime is determined at creation by the existing backlog in the source subscription. The lifetime of the snapshot is seven days, which is the age of the oldest unacked message in the subscription. Consider a subscription with an oldest unacked message that is three days old. If a snapshot is created from this subscription, the snapshot, which always captures this three day old backlog as long as the snapshot exists, will expire in four days. The service will refuse to create a snapshot that will expire in less than one hour after creation. A timestamp is in RFC3339 UTC Zulu format, with nanosecond resolution and up to nine fractional digits, such as 2014-10-02T15:01:23Z and 2014-10-02T15:01:23.045123456Z. |  |
+| Expire Time a| DateTime |
+
+Time that the snapshot is guaranteed to exist up until. A newly-created snapshot
```

Comment:
The parenthetical expression confuses things. Can you write something like this? (I also reworded the first sentence a bit to make it less awkward.)

Time length for which the snapshot is guaranteed to exist. The lifetime for a newly-created snapshot is based on the oldest unacked message in the source subscription's backlog, however, the snapshot can exist for a maximum of seven days.

Consider...